README.md

```diff
@@ -307,12 +309,15 @@ provided by this library:
 
 ## Testing Suggestors
 
+_TODO: create a `codemod_test` package that exports `applySuggestor()` and
+`fileContextForTest()` and update the following docs_
```

Comment:
I'd like to export the `fileContextForTest()` method so that testing suggestors is still just as easy as it is now (which only requires creating a `SourceFile.fromString()`, but my current implementation depends on `package:test_descriptor` to create the files in a temporary directory and clean them up during teardown.. so I'm thinking we could add a `codemod_test` package to this repo that just exports:
- `applySuggestor()`
- `fileContextForTest()`

Reply:
I don't think it is necessary to separate it out. Seems like codemod or packages that use codemod are dev-dependencies or globally activated packages, so I don't think it should be a big deal to add that dependency in this package.

Reply:
Agreed, I decided to just add a `test.dart` entrypoint